src/dev/src/app/custom-props/custom-props.demo.module.ts

```diff
@@ -12,7 +12,7 @@ import { ClarityModule } from '@clr/angular';
 import { CustomPropsDemo } from './custom-props.demo';
 import { ROUTING } from './custom-props.routing';
 
-import '@clr/base/components/clr-wc-element';
+import '@clr/base/clr-wc-element';
```

Comment:
You don't want the components to live in their own folder like helpers and utils?

Reply:
Updated to use `/components` directory and path